### ISSUE TITLE
Change description of TEI Header and simplify content model of TEI

### ISSUE THREAD

**Issue by TEITechnicalCouncil** [TEIC/TEI#561]
It seems time to rethink the definitions of the teiHeader element in
line with expanded applications of it as a generic metadata
container. At the same time, it would be useful to simplify the
content model of the &lt;TEI> element itself.

This ticket proposes the following changes:
1. Change the description of &lt;teiHeader> from "supplies the
   descriptive and declarative information making up an electronic title
   page for every TEI-conformant document."  to "supplies descriptive and
   declarative metadata associated with a digital resource or set of
   resources". This change makes it easier to use the teiHeader to
   document an &lt;fsdDecl> (as is currently proposed) and also to use it
   with the proposed new &lt;standOff> element.
2. Change the description for the class model.resourceLike from
   "groups non-textual elements which may appear together with a header
   and a text to constitute a TEI document" to "groups separate elements
   which constitute the content of a digital resource, as opposed to its
   metadata."
3. Change the content model of &lt;TEI> from 

```
  <content>
    <group xmlns="http://relaxng.org/ns/structure/1.0">
      <ref name="teiHeader"/>
      <choice>
        <group>
          <oneOrMore>
            <ref name="model.resourceLike"/>
          </oneOrMore>
          <optional>
            <ref name="text"/>
          </optional>
        </group>
        <ref name="text"/>
      </choice>
    </group>
  </content>
```

to

```
  <content>
    <group xmlns="http://relaxng.org/ns/structure/1.0">
      <ref name="teiHeader"/>
      <choice>
          <oneOrMore>
            <ref name="model.resourceLike"/>
          </oneOrMore>
      </choice>
    </group>
  </content>
```

and (of course) add `<text>` to model.resourceLike

The only possible downside of this change is that if you have an ODD
in which model.resourceLike is deleted for some reason, you will now
thereby delete the &lt;text> element. But deleting model classes is an
intrinsically dangerous procedure which should only be undertaken by
the fully aware.

Original comment by: @lb42

**Comment by martindholmes** [comment]
Council subgroup 2015-10-26: Make it green, someone to go ahead and test this in a branch and report back.

**Comment by bansp** [comment]
This nice proposal has escaped my attention, until today's happy finale. Thanks! :-)

**Comment by lb42** [comment]
Not sure I understand how to do the required gittery, but I have tested this in my local copy and it all looks fine. The desc for TEI also needs changing though:

"contains a single  TEI-conformant document, combining a single TEI header with one or  
more members of the model.resourceLike class. Multiple <gi>TEI</gi> elements may be combined  to
form a <gi>teiCorpus</gi> element"

**Comment by lb42** [comment]
Closed, as now implemented

**Comment by lb42** [comment]
Reopening, since the implementation was incomplete. The prose of DS needs checking more carefully. More significantly, the content model of TEI should  be revised to permit only single occurrences of any member of model.resourceLike, i.e. using the attribute @expand="sequenceOptional" . Some documentation should be added to remind us what the required sequence is (it's the order in which the children are declared, from memory, but no-one could be expected to know that). This should have been done when the content model was purified.

**Comment by sydb** [comment]
I think that proposal is the right idea, @lb42. However, we would need to think through the order of elements. Currently it would resolve to
~~~~~
  ( teiHeader, fsdDecl?, \text?, facsimile?, sourceDoc? )
~~~~~
But we have one example each of `<facsimile>` and `<sourceDoc>` before `<text>`.

**Comment by jamescummings** [comment]
Council Subgroup (Victoria 2017) thinks that a change is not necessary but if it was then it would need a period of deprecation since it would break backwards compatibility.  We agree that there are some prose that need to be changed, especially in DS, and appreciate any pointers to them that readers of this issue come across.

**Comment by sydb** [comment]
So this ticket is left open for tweaks to the prose of the first few paras of DS.

**Comment by sydb** [comment]
Closed at 136088a. I don’t claim for a moment that the wording could not be improved, only that it is better now than it was before.

**Comment by lb42** [comment]
Sorry, but this is just Wrong. If the content model of `<TEI>` permits multiple `<text>` children, what is `<teiCorpus>` for ? Syd's rewording  makes explicit that a TEI can contain multiple text elements, which is indeed what the current content model permits, but fixing that craziness was my purpose in raising this issue! The only argument for leaving it so seems to be a fear of breaking compatibility, but (a) introducing this possibility was done comparatively recent (b) retaining compatibility with a bad decision is not necessarily a good idea.
